oma-console/src/print.rs

```diff
@@ -116,7 +134,7 @@ fn term_color<D>(input: D, color: Action) -> StyledObject<D> {
         Action::PendingBg => style(input).bg(Color::Blue).bold(),
     }
 }
-
+/// OmaLayer
```

Comment:
`OmaLayer` and `OmaRecorder` are used to output oma-style logs to `tracing`, such as in `src/main.rs:206`:

```rust
    if !debug {
        let no_i18n_embd_info: EnvFilter = "i18n_embed=off,info".parse().unwrap();

        tracing_subscriber::registry()
            .with(
                OmaLayer
                    .with_filter(no_i18n_embd_info)
                    .and_then(LevelFilter::INFO),
            )
            .init();
    } 
```

Reply:
Fixed this in https://github.com/AOSC-Dev/oma/pull/228/commits/2d03568ca097c37f4812813b1688a6d68434c17c